docs/swarm/key-concepts.md

```diff
@@ -10,77 +10,79 @@ parent="engine_swarm"
 weight="2"
 +++
 <![end-metadata]-->
-# Docker Swarm key concepts
+# Swarm mode key concepts
 
-Building upon the core features of Docker Engine, Docker Swarm enables you to
-create a Swarm of Docker Engines and orchestrate services to run in the Swarm.
-This topic describes key concepts to help you begin using Docker Swarm.
+This topic introduces some of the concepts unique to the cluster management and
+orchestration features of Docker Engine 1.12.
 
 ## Swarm
 
-**Docker Swarm** is the name for the cluster management and orchestration
-features embedded in the Docker Engine. Engines that are participating in a
-cluster are running in **Swarm mode**.
+The cluster management and orchestration features embedded in the Docker Engine
+are built using **SwarmKit**. Engines participating in a cluster are
+running in **Swarm mode**. You enable Swarm mode for the Engine by either
+initializing a Swarm or joining an existing Swarm.
 
-A **Swarm** is a cluster of Docker Engines where you deploy a set of application
-services. When you deploy an application to a Swarm, you specify the desired
-state of the services, such as which services to run and how many instances of
-those services. The Swarm takes care of all orchestration duties required to
-keep the services running in the desired state.
+A **Swarm** is a self-organizing cluster of Docker Engines where you deploy
+[services](#Services-and-tasks). The Docker Engine CLI includes the commands for
+Swarm management, such as adding and removing nodes. The CLI also includes the
+commands you need to deploy services to the Swarm and manage service
+orchestration.
 
-## Node
+When you run Docker Engine outside of Swarm mode, you execute container
+commands. When you run the Engine in Swarm mode, you orchestrate services.
```

Comment:
I think either here or in the overview, we should draw a distinction to legacy "Swarm". This is going to be a FAQ and a constant source of confusion.